openedx/features/learner_analytics/views.py

```diff
@@ -97,3 +109,84 @@ def get_schedule(self, request, course_key):
             if all_blocks['blocks'][block].get('graded', False):
                 graded_blocks[block] = all_blocks['blocks'][block]
         return json.dumps(graded_blocks)
+
+    def get_weekly_course_activities(self, course_key):
+        """
+        Get the count of any course activity from previous 7 days
+
+        Args:
+            course_key: CourseKey
+        """
+        cache_key = 'learner_analytics_{course_key}_weekly_activities'.format(course_key=course_key)
+        activities = cache.get(cache_key)
+
+        if not activities:
+            log.info('Weekly course activities for course {course_key} was not cached - fetching from Analytics API'
+                     .format(course_key=course_key))
+            activities = self.analytics_client.courses(course_key).activity()
+
+            seconds_to_cache_expiration = LearnerAnalyticsView.seconds_to_cache_expiration()
+            log.info('Weekly course activities cache expiration: {seconds_to_cache_expiration}'
+                     .format(seconds_to_cache_expiration=seconds_to_cache_expiration))
+            cache.set(cache_key, activities[0], seconds_to_cache_expiration)
+            activities = activities[0]
+
+        return activities['any']
+
+    @staticmethod
```

Comment:
Is there a reason why this is a static method?

Reply:
My PyCharm was complaining about this but I can remove this and see if it affects quality.